cli/src/main/scala/com/advancedtelematic/tuf/cli/CommandHandler.scala

```diff
@@ -71,7 +87,7 @@ object CommandHandler {
 
     case InitTargets =>
       tufRepo
-        .initTargets(config.version.valueOrConfigError, config.expires)
+        .initTargets(config.version.valueOrConfigError, config.expireOn.getOrElse(Instant.now().plus(DEFAULT_TARGET_LIFETIME)))
```

Comment:
why not using `expirationDate` as well?

Reply:
Because the ticket was only about signing, I didn't want to introduce new functionality randomly.